frame/staking/src/lib.rs

```diff
@@ -435,36 +434,35 @@ pub struct UnlockChunk<Balance: HasCompact> {
 	value: Balance,
 	/// Era number at which point it'll be unlocked.
 	#[codec(compact)]
-	era: EraIndex,
+	unlock_era: EraIndex,
```

Comment:
Renamed this for readability

Reply:
I was originally for this change, but since it breaks all UI/apps, I think we should revert. I mean, the parent struct is called `*Unlock*Chunk`, I guess that's good enough.